dev_requirements/requirements-generate-bo4e.txt

```diff
@@ -10,7 +10,7 @@ argcomplete==3.1.6
     # via datamodel-code-generator
 black==23.12.1
     # via datamodel-code-generator
-bo4e-python-generator==0.0.4
+bo4e-python-generator==0.0.5
```

Comment:
kannst du in der entsprehcenden requirements.in auch die min-version setzen?

Reply:
https://github.com/Hochfrequenz/intermediate-bo4e-migration-models/pull/110/commits/770dcf360d4e61bf34cfce776668fe96d818327f